File: API/Shopking/Shopking/Models/Recipe.cs
using System;
using System.Collections.Generic;

namespace Shopking.Models
{
    public class Recipe
    {
        public virtual long Id { get; set; }
        public virtual string Name { get; set; }
        public virtual RecipeDifficulty Difficulty { get; set; }
        public virtual IList<Ingredient> Ingredients { get; set; }
        public virtual int EstimatedTime { get; set; }
        public virtual string Description { get; set; }

        public Recipe()
        {
        }
    }
}

File: Android/Shopking/app/src/main/java/hr/fer/objobl/shopking/data/model/Recipe.kt
package hr.fer.objobl.shopking.data.model

data class Recipe(
    val id: Long,
    val name: String,
    val description: String,
    val imageUrl: String,
    val ingredients: List<String>,
    val timeEstimated: Int,
    val totalPrice: String,
    val difficulty: String
)
